CHANGELOG.md

```diff
@@ -24,6 +24,7 @@ NOTE: [`eframe`](eframe/CHANGELOG.md), [`egui_web`](egui_web/CHANGELOG.md) and [
   enabling zoom, rotate, and more. Works with `egui_web` on mobile devices,
   and should work with `egui_glium` for certain touch devices/screens.
 * Add (optional) compatibility with [mint](https://docs.rs/mint).
+* `TextEdit` now supports edits on a generic buffer using `TextBuffer`.
```

Comment:
This should be under the `Unreleased` heading!

Reply:
Oops, I somehow missed that heading. Should be fixed now.